lesson5.html

```diff
@@ -0,0 +1,45 @@
+<html>
+    <head>
+        <title>Mouse leave and over event</title>
+        <link rel="stylesheet" type="text/css" href="style5.css">
+    </head>
+    <body>
+        <div id ="newBtn"></div>
+        <div class="btn-postn"><button id ="awsome">Just-Awsome</button></div>
+        
+        <script>
+            let awsmBttnCmp = document.getElementById("awsome");
+            let newCmp = document.getElementById("newBtn");
+            const label = "Im Awsome" ;
+             function onClick(event){
+                 console.log("Hey!! Be positive");
+             }
+             function mouseOverHandler (event){
+                 console.log("You are here");
+                 awsmBttnCmp.innerText = "Be Awsm" ;
+                 
+                 awsmBttnCmp.classList.add("button-shape");
+                 let bttnNew = document.createElement("button");
+                 awsmBttnCmp.classList.add("button-hover");
+                 awsmBttnCmp.classList.remove("button-original");
+                 bttnNew.innerText = "Stay Calm" ;
+                 newCmp.appendChild(bttnNew); 
+                 p.style.display="block";
+
+
+             }
+             function mouseLeaveHandler (event){
+                 console.log("uh oh!! You just left");
+                 awsmBttnCmp.innerText = label ;
+                 awsmBttnCmp.classList.add("button-original");
+                 awsmBttnCmp.classList.remove("button-hover");
+                 awsmBttnCmp.classList.remove("button-shape");
+                 newCmp.removeChild(bttnNew);
```

Comment:
bttnNew is not accessible in this function, as it was introduced in another function. To make it available you can do so:
1. In script itself (right under let newCmp = document.getElementById("newBtn");) introduce variable : let bttnNew;
2. In mouseOverHandler function remove let infront of bttnNew: bttnNew = document.createElement("button");
3. Now you can use it in other functions.